club/style.css

```diff
@@ -63,15 +68,7 @@ a {
 	.wp-block-file .wp-block-file__button,
 	.wp-block-button__link,
 	.wp-elements-button,
-):where(:active) {
-	border-style: dotted;
-}
-
-:is(
-	.wp-block-search__button,
-	.wp-block-file .wp-block-file__button,
-	.wp-block-button__link,
-	.wp-elements-button,
+	.wp-block-read-more,
 ):where(:active) {
 	border-style: dotted;
```

Comment:
Only a small detail, but I think this should be `dashed` rather than `dotted`, when looking at the comps.

Reply:
good catch! Change ready!